nixos/doc/manual/development/writing-nixos-tests.section.md

```diff
@@ -1,9 +1,9 @@
 # Writing Tests {#sec-writing-nixos-tests}
 
-A NixOS test is a Nix expression that has the following structure:
+A NixOS test is a Nix module that has the following structure:
```

Comment:
```suggestion
A NixOS test is a NixOS module that has the following structure:
```

Oh boy, I just realized the NixOS manual _still_ does not have a proper definition of modules. The good one we have is [NixOS Wiki - Module](https://nixos.wiki/wiki/Module).

Reply:
I don't think we should call a module a NixOS module if it doesn't mix with actual NixOS modules. Calling it a "Nix module" (like I did) is also wrong. I'll use just "module".

Reply:
Good point.